### ISSUE TITLE
Add visible focus to Explore and Query functions

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#2752]
As per Trusted Tester, A visible indication of focus is provided when focus is on the interface component.

[Query Data](https://revenuedata.doi.gov/query-data) & [Explore Data](https://revenuedata.doi.gov/explore?dataType=Revenue&location=NF%2CNA&mapLevel=State&offshoreRegions=false&period=Calendar%20Year&year=2022)

- Year slider (this should be fixed with issue #2748)[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBczhFIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--36eb5c8ad52fb84706e01694611923fd4c07907a/image.png)

- Filter options for checkbox dropdowns do not receive focus as you move through the list [Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdE1FIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--2dad24cfeec05ddfcfaaffffe6c89025119d5b28/image.png)

[Explore Data](https://revenuedata.doi.gov/explore?dataType=Revenue&location=NF%2CNA&mapLevel=State&offshoreRegions=false&period=Calendar%20Year&year=2022)

- Offshore toggle [Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdGNFIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--c8ead204522221600fb1f82ed1e17bed886151dd/image.png)

- Comparison and nationwide summary button gets focus but it's barely visible

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdVFFIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--81dd4987d02051e303c54db552e9cdd298cec54b/image.png)

**Comment by lpgoldstein** [comment]
No longer relevant
